doc/new-guidelines.md

```diff
@@ -0,0 +1,468 @@
+# Guidelines
+
+>The goal of this document is to help all contributors (core and external) to
+>write code in _unison_ and help establish good practices that serve the Status
+>Mobile contributors well.
+
+We don't want to turn this document into an exhaustive list of rules to be
+followed that nobody will read. As much as possible, we'll try to document only
+what we consider important for Status Mobile. In other words, we don't want to
+maintain a general Clojure convention/style guide, nor do we want to turn this
+document into a long tutorial.
+
+This is a **work in progress**, and not all conventions are properly implemented
+in the codebase yet. The project structure is also going over major changes (as
+of Nov/2022), and it'll take a considerable amount of time until we migrate the
+existing code to the new structure.
+
+If you find out anything is outdated or missing, please, share with us or even
+better, create a pull-request! 🤸
+
+## Style guide
+
+We mostly follow the [Clojure Style
+Guide](https://github.com/bbatsov/clojure-style-guide), so it's recommended to
+get familiar with it.
+
+As of Nov/2022, running `make lint` should fix the most basic formatting issues,
+and we are in the process of integrating a tool to format non-trivial code
+according to the Clojure Style Guide. This should greatly reduce noise in
+pull-request reviews and it'll simplify the life of all contributors.
+
+Pay special attention to:
+
+- Align let bindings https://github.com/bbatsov/clojure-style-guide#bindings-alignment
+- Align map keys https://github.com/bbatsov/clojure-style-guide#map-keys-alignment
+
+## Dos and don'ts
+
+### Component styles
+
+Prefer to define styles in separate files named `style.cljs`, usually colocated
+with the source file using it. For an example, see
+[src/quo/components/controls/styles.cljs](../src/quo/components/controls/styles.cljs).
```

Comment:
Update `styles.cljs` to `style.cljs` here and in the usage example too

Reply:
Actually, if it is a real example and we decide to use `style.cljs`, we should update the example in our code to avoid confusions

Reply:
We come back to the documentation and update it as soon as we have real examples using singular file names.

Reply:
Actually there are style files named in the singular form. I'll update the docs to point to them.

Reply:
Done, thanks!